src/main/java/com/chessgrinder/chessgrinder/chessengine/JavafoPairingStrategyImpl.java

```diff
@@ -21,13 +22,9 @@ public class JavafoPairingStrategyImpl implements PairingStrategy {
      * JaVaFo is no thread-safe library, therefore to avoid concurrency problems,
      * the requests to JaVaFo are synchronized on this monitor.
      */
-    private static final Object JAVAFO_MONITOR = new Object();
 
     private final PlayerTrfLineParser playerTrfLineParser = new PlayerTrfLineParser();
     private final MissingPlayersTrfLineParser missingPlayersTrfLineParser = new MissingPlayersTrfLineParser();
-    private static final int DEFAULT_RATING = 1000;
-    private static final String NEWLINE_REGEX = "\\r?\\n|\\r";
-    private static final int STANDARD_PAIRING_CODE = 1000;
```

Comment:
no, please, these constants belong here. do not move them out of here.

Reply:
da blyat'